### PR TITLE
epic/statistics-7 tests fixups

### PR DESCRIPTION
**Pull request by fiedukow**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java [hunk 1]
```diff
@@ -219,7 +219,8 @@ private PlanNodeStatsEstimate symbolToSymbolEquality(Symbol left, Symbol right)
             SymbolStatsEstimate leftStats = input.getSymbolStatistics().get(left);
             SymbolStatsEstimate rightStats = input.getSymbolStatistics().get(right);
 
-            if (isNaN(leftStats.getDistinctValuesCount()) || isNaN(rightStats.getDistinctValuesCount())) {
+            if (leftStats == null || rightStats == null ||
```
Comment (losipiuk) [inline presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java]: If you use `input.getSymbolStatistics(left)` instead `input.getSymbolStatistics().get(left)` you will not need to check for null.

File: presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java [hunk 2]
```diff
@@ -151,6 +151,10 @@ private PlanNodeStatsEstimate symbolToLiteralGreaterThan(Symbol symbol, double l
         {
             SymbolStatsEstimate symbolStats = input.getSymbolStatistics().get(symbol);
 
+            if (symbolStats == null) {
```
Comment (losipiuk) [inline presto-main/src/main/java/com/facebook/presto/cost/FilterStatsCalculator.java]: same comment here.

**Review by losipiuk — CHANGES_REQUESTED** [review]

File: presto-main/src/main/java/com/facebook/presto/cost/SymbolStatsEstimate.java [hunk 3]
```diff
@@ -52,6 +50,11 @@ public double getHighValue()
         return highValue;
     }
 
+    public boolean hasNonNullValues()
+    {
+        return distinctValuesCount != 0 || isNaN(lowValue) || isNaN(highValue);
```
Comment (losipiuk) [inline presto-main/src/main/java/com/facebook/presto/cost/SymbolStatsEstimate.java]: distinctValuesCount != 0 || (!isNaN(lowValue) && !isNaN(highValue));

**Comment by fiedukow** [comment]
merged into epic/statistics-9

**Pull request closed without merging**